nncf/layers.py

```diff
@@ -95,6 +104,9 @@ def from_module(module):
 
 
 class NNCFConvTranspose3d(_NNCFModuleMixin, nn.ConvTranspose3d):
+    op_func_name = "conv_transpose3d"
```

Comment:
If op_func_name is a required attribute for all NNCF-wrapped modules now, it should be located in a base class. It should also be documented in code there.